### PR TITLE
pd: add note for member `leader_priority`

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -705,6 +705,25 @@ member leader transfer pd3
...
+member leader_priority  pd-4 1
+member leader_priority  pd-5 0
+```
+```
+......
+```
+
+> **注意：**
+>
+> 优先级数值越大的节点成为 leader 的可能性越高。
```
Comment (okJiang) [inline pd-control.md:725]: 看代码数值最大的直接会变成 leader？

Comment (okJiang) [reply pd-control.md:725]: https://github.com/tikv/pd/blob/f0c84e49b4aea4104ff4151b2abdc9712a6b2db7/pkg/member/member.go#L300

Comment (lhy1024) [reply pd-control.md:725]: 是的，不过没太想好表述，就复制之前的文档里的了。“在可用的节点中，数值最大的节点加入集群后，会直接当选 leader”，这样好一些？

Comment (okJiang) [reply pd-control.md:725]: 在可用的节点中，数值最大的节点会直接当选 leader

我觉得这样就行，可以让文档同学来看，意思对了就行

**Review by okJiang — APPROVED** [review]

File: dr-multi-replica.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ summary: 了解 TiDB 提供的基于多副本的单集群容灾方案。
 
     > **注意：**
     >
-    > 优先级数值越大的节点成为 leader 的可能性越高。
+    > 在可用的节点中，数值最大的节点会直接当选 leader
```
Comment (qiancai) [inline dr-multi-replica.md:122]:
```suggestion
    > 在可用的 PD 节点中，优先级数值最大的节点会直接当选 leader。
```

File: pd-control.md [hunk 3]
```diff
@@ -705,6 +705,25 @@ member leader transfer pd3
 ......
 ```
 
+指定 PD leader 的优先级:
+
+{{< copyable "" >}}
+
```
Comment (qiancai) [inline pd-control.md:711]:
```suggestion
```

File: pd-control.md [hunk 4]
```diff
@@ -705,6 +705,25 @@ member leader transfer pd3
...
+member leader_priority  pd-4 1
+member leader_priority  pd-5 0
+```
+```
+......
+```
+
+> **注意：**
+>
+> 在可用的节点中，数值最大的节点会直接当选 leader
```
Comment (qiancai) [inline pd-control.md:725]:
```suggestion
> 在可用的 PD 节点中，优先级数值最大的节点会直接当选 leader。
```

File: three-data-centers-in-two-cities-deployment.md [hunk 5]
```diff
@@ -196,7 +196,7 @@ tikv_servers:
     >
     > TiDB 5.2 及以上版本默认不支持 `label-property` 配置。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md)。
 
-- 设置 PD 的优先级，为了避免出现跨区域 AZ 的 PD 成为 Leader，可以将本区域 AZ 的 PD 优先级调高（数字越大，优先级越高），将跨区域的 PD 优先级调低。
+- 设置 PD 的优先级，为了避免出现跨区域 AZ 的 PD 成为 Leader，可以将本区域 AZ 的 PD 优先级调高（数字越大，优先级越高），将跨区域的 PD 优先级调低。在可用的节点中，数值最大的节点会直接当选 leader。
```
Comment (qiancai) [inline three-data-centers-in-two-cities-deployment.md:199]:
```suggestion
- 设置 PD 的优先级，为了避免出现跨区域 AZ 的 PD 成为 Leader，可以将本区域 AZ 的 PD 优先级调高（数字越大，优先级越高），将跨区域的 PD 优先级调低。在可用的 PD 节点中，优先级数值最大的节点会直接当选 leader。
```

File: pd-control.md [hunk 6]
```diff
@@ -705,6 +705,23 @@ member leader transfer pd3
...
+```bash
+member leader_priority  pd-1 4
+member leader_priority  pd-2 3
+member leader_priority  pd-3 2
+member leader_priority  pd-4 1
+member leader_priority  pd-5 0
+```
+```
+......
+```
```
Comment (qiancai) [inline pd-control.md:719]:
````suggestion
```
......
```
````

**Review by qiancai — APPROVED** [review]

**Pull request merged**